src/platform/icelake/include/platform/drivers/alh.h

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I guess we need it also for tgl

Reply:
In a separate PR, likely requires different handshakes.